npm/pkg/dataplane/ipsets/ipsetmanager_test.go

```diff
@@ -1509,3 +1509,55 @@ func assertExpectedInfo(t *testing.T, iMgr *IPSetManager, info *expectedInfo) {
 		require.Equal(t, expectedNumEntries, numEntries, "numEntries mismatch for set %s", set.Name)
 	}
 }
+
+func TestValidateIPSetMemberIP(t *testing.T) {
```

Comment:
nit: let's put this code next to the other validate ip block tests

Reply:
done